src/mangosd/mangosd.conf.dist.in

```diff
@@ -383,7 +383,7 @@ CleanCharacterDB                  = 1
 
 LogSQL                       = 1
 PidFile                      = ""
-LogLevel                     = 3
+LogLevel                     = 1
```

Comment:
Is there some reason to change this?
If yes, then the default should be changed in the documentation above too.